resources/assets/js/components/Leaderboard.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
Any specific reason why you chose to you with a Vue component instead of a using a simple `foreach` in blade? - Just Curious!

Reply:
Actually, I started with blade and migrated to Vue although I don't have a strong preference either way. 

Having said that, I do like the pattern whereby the back-end is mostly api + just enough blade to load the page with the main UI heavy lifting is done Vue. I find it easier to reason about what is presentational logic and syntactic sugar and therefore what to test. 

Admittedly, in this case we are talking about a very simple feature and either blade based or Vue based would be fine. In the end I just went with what I've been doing lately.